### PR TITLE
Add pyjwt to pyproject

### PR DESCRIPTION
**Pull request by rbren**
**What is the problem that this fixes or functionality that this introduces? Does it fix any open issues?**

I tried to set up from scratch today, and hit an error with `jwt` missing.

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

Adds `pyjwt` to the pyproject.toml

---
**Other references**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -41,6 +41,7 @@ grep-ast = "0.3.2"
 tree-sitter = "0.21.3"
 bashlex = "^0.18"
 
+pyjwt = "^2.9.0"
```
Comment (xingyaoww) [inline pyproject.toml:44]:
```suggestion
pyjwt = "^2.9.0"

```

**Review by tobitege — APPROVED** [review]
LGTM, wondering though if this was committed with pre-commit or without?

**Review by xingyaoww — APPROVED** [review]

**Comment by rbren** [comment]
@tobitege I think I used the standard precommit...

No idea why the tests are failing

**Comment by tobitege** [comment]
The pyproject.toml file needs "build" added.

**Comment by tobitege** [comment]
@rbren I've added build to the pyproject.toml file, could you pull this and do a `poetry lock --no-update`, please?

**Comment by rbren** [comment]
Done!

**Pull request merged**